src/game_launcher.cpp

```diff
@@ -831,23 +835,26 @@ void game_launcher::start_wesnothd()
 		filesystem::write_file(config, filesystem::read_file(filesystem::get_wml_location("lan_server.cfg")));
 	}
 
+	LOG_GENERAL << "Starting wesnothd";
+	try
+	{
 #ifndef _WIN32
-	std::string command = "\"" + wesnothd_program +"\" -c \"" + config + "\" -d";
+		bp::child c(wesnothd_program, "-c", config);
```

Comment:
Wasn't the Windows version supposed to have some extra flag or something…?

Reply:
Yeah, but it failed to build, so I'm seeing if it builds at all without the special flag.

Reply:
Is the `-d` no longer needed?

Reply:
Detaching the child process is equivalent, I think?

Reply:
Yes.